acestep/core/generation/handler/init_service_loader.py

```
@@ -0,0 +1,212 @@
+        *,
+        model_checkpoint_path: str,
+        device: str,
+        use_flash_attention: bool,
+        compile_model: bool,
+        quantization: Optional[str],
+    ) -> str:
+        """Load DiT, apply compile/quantization options, and return selected attention backend."""
+        from transformers import AutoModel
+
+        if not os.path.exists(model_checkpoint_path):
+            raise FileNotFoundError(f"ACE-Step V1.5 checkpoint not found at {model_checkpoint_path}")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)